### PR TITLE
Update Firefox Privacy Notice with new branding (#6858)

### LINKED ISSUE #6858: Update Firefox Privacy Notice with new branding
## Description

Update Firefox Privacy Notice with new branding

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Page reflects new branding
- [ ] Other pages generated from `process_legal_doc()` are not broken

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

- Change the way the markdown is processed to remove `<header>`s for sections
  - Update other page templates to keep accordion widgets working with new markup
- Change styles for /privacy/firefox and /privacy/firefox-lite
  - Including the data-choices widget
- Update JS for firefox and firefox-lite pages to rely on Protocol functions already in common JS

## Issue / Bugzilla link

Fix #6858

## Testing
- On [demo4](https://www-demo4.allizom.org/en-US/privacy/firefox/)
- You will need to give UITour permission to run before you can test the data-choices widget

### CODE REVIEW

**Comment by stephaniehobson** [comment]
@dzingeek Ready for design review.

**Comment by dzingeek** [comment]
@stephaniehobson this looks great! I just have 2 small tweaks.

**Legal copy at the bottom (e.g. "This privacy notice...")**
1. Remove grey background color block. I don't think it's needed. 
2. Please make this section of copy 16px instead of the typical 18px

**Comment by stephaniehobson** [comment]
Design tweaks made. Ready for review!

File: bedrock/privacy/templates/privacy/base-protocol.html [hunk 1]
```diff
@@ -30,23 +28,17 @@
...
-<article class="section-content mzp-t-firefox" itemtype="http://schema.org/Article">
-  <header class="privacy-head">
-    <div class="l-narrow">
-      <div class="policy-title">
-        <h1 itemprop="name">{{ header.h2|join|safe }}</h1>
-        <time datetime="{{ header.select('[datetime]')[0]['datetime'] }}" itemprop="dateModified">{{ header.select('[datetime]')[0].string }}</time>
-      </div>
+<article class="section-content mzp-l-content mzp-t-firefox" itemtype="http://schema.org/Article">
+  <div class="l-narrow">
+    <div class="policy-title">
```
Comment (amychurchwell) [inline bedrock/privacy/templates/privacy/base-protocol.html]: Nit: should we keep this as a header element?

Comment (stephaniehobson) [reply bedrock/privacy/templates/privacy/base-protocol.html]: Added the headers back. Thanks!

**Pull request merged**